portality/upgrade.py

```diff
@@ -55,10 +55,7 @@ def do_upgrade(definition, verbose):
             tasks = tdef.get("tasks", None)
             if tasks:
                 for func_call, kwargs in tasks.iteritems():
-                    if args:
```

Comment:
can you tell what this used to refer to? I can't find an args variable, there's just kwargs...